frontend/src/app/components/dashboard/db-table-view/db-table-view.component.css

```
@@ -2,6 +2,43 @@
+	}
+
+	.pull-refresh-indicator__icon_spinning {
+		animation: pullRefreshSpin 700ms linear infinite;
+	}
+
+	.refresh-button {
+		display: none !important;
+	}
+}
+
+@keyframes pullRefreshSpin {
```

Stylelint (17.12.0)

[error] Expected keyframe name "pullRefreshSpin" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)